hw/ip/pwm/data/pwm_testplan.hjson

```diff
@@ -2,111 +2,224 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  name: "pwm"
-  import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/enable_reg_testplan.hjson",
-                     // TODO: enable for V2
-                     //"hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"],
-  testpoints: [
-    {
-      name: smoke
-      desc: '''
-            Smoke test accessing a major datapath within the pwm
+    name: "pwm"
+    import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
+                       "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
+                       "hw/dv/tools/dvsim/testplans/enable_reg_testplan.hjson",                      
+                       "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson",
+                       "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"],
+    testpoints: [
+        {
+            name: smoke
+            desc: '''
+            Smoke is a simple test accessing a major data path within the pwm
 
             Stimulus:
-              - Program pwm enable register (PWM_EN)
-              - Program configuration registers (CFG, INVERT)
-              - Disable blink and heartbeat enable registers (these mode are verified with other tests)
-              - Program REGEN to lock (enable) register settings and start pwm function
+            - run segments on one channel in standard mode
+            - both send read and write commands
+            - reads and writes the Host FIFO's
+                - uses different clock divisions and resolutions
```

Comment:
Did this get accidentally added from SPI_HOST?  What are these fifos?  Read and write commands? Segments?  I think the previous stimulus makes much more sense.

Reply:
I have no clue what happended - I was task switching between the do a few days back - must have accidentally thought I was doing spi but still that would also not make sense. I have updated it to what it should be